src/middleware/translations/en.json

```diff
@@ -211,18 +211,15 @@
 	},
     "routes": {
         "healthcheck": "healthcheck",
-        "view": {
-            "start": "dataset"
-        },
-        "publish": {
-            "start": "publish",
-            "title": "title",
-            "upload": "upload",
-            "preview": "preview",
-            "confirm": "confirm",
-            "sources": "sources",
-            "source_confirmation": "source-confirmation",
-            "tasklist": "tasklist"
-        }
+        "dataset": "dataset",
+        "publish": "publish",
+        "start": "start",
+        "title": "title",
+        "preview": "preview",
+        "upload": "upload",
+        "confirm": "confirm",
+        "sources": "sources",
+        "source_confirmation": "source-confirmation",
+        "tasklist": "tasklist"
```

Comment:
Technically these should never get used as the url builder skips attempting to translate URLs when the locale is English.

However, we'll need them as fallback if the Welsh is missing, and the Welsh language translations need the right keys in order to work, so it makes sense for the English translation file to have them as a source.

Reply:
PS - these must be flat - the builder translates each bit of the path individually not as a whole.